manifest.yml

```diff
@@ -23,13 +23,13 @@ url_to_dependency_map:
 dependencies:
   - name: openresty
     version: 1.9.3.1
-    uri: https://raw.githubusercontent.com/muymoo/staticfile-buildpack/files/vendor/openresty-1.9.3.1.tar.gz
+    uri: https://raw.githubusercontent.com/sks/staticfile-buildpack/files/vendor/openresty-1.9.7.3.tar.gz
```

Comment:
If you are to change this , then you might want to pull in the latest "file" branch from my repo

https://github.com/sks/staticfile-buildpack/tree/files